tests/src/JIT/HardwareIntrinsics/X86/Avx/Add.Double.cs

```diff
@@ -28,32 +28,41 @@ private static void AddDouble()
                 // Validates basic functionality works, using Unsafe.Read
                 test.RunBasicScenario_UnsafeRead();
```

Comment:
It would be helpful to separate all template changes which impact AVX, AVX2, SSE and SSE2 into separate PR as they are unrelated to to implementation of SSE2 and SSSE3 intrinsics. IMO change of template for generating tests which impact multiple ISAs should be always separated from implementation and tests for a particular ISA like in this case.

Reply:
The changes which impact existing ISAs are required to implement SSE3 and SSSE3 intrinsics. Due to the way template generation works, it also requires regenerating the other templates. As a compromise, the modification of the existing templates and regeneration of the existing tests has been split out into their own commit.

Additionally, given that these are all generated from a template, it is not necessary to examine every single change. Rather, it is only required to examine the changes made to the template and the changes made to one of each test that was generated from said template (which cuts down the actual changed code to less than 500 lines, rather than 20k lines).

Reply:
To be clear, I would normally agree, especially if such a change was being blocked by work which isn't yet complete. However, given that this PR, as a whole, is complete; Splitting it out, waiting for tests to rerun, and waiting for the review to complete, rebasing this PR after the merge, waiting for those tests to rerun, and waiting for that review to complete, may actually take more time than it would to finish reviewing the ~500 changed lines in this PR and getting them merged simultaneously.

Reply:
> To be clear, I would normally agree

I understand your point, however, when someone would read git history, it would be very hard to understand scope of the change based only on titles of commits. Perhaps making more precise commit title which introduces test template changes with some more expanded description would be  helpful and would better align with coding guidelines which specifically say how and what to write in git commit titles and descriptions.

Reply:
@CarolEidt, do you have a preference on whether or not this is split into two PRs? If it was, this PR would be blocked by the PR containing just the template adjustments.

Reply:
I don't feel that it needs to be split. I will try to get it reviewed this evening.